docs/gameserverapi/apidocs.md

```diff
@@ -0,0 +1,775 @@
+---
```

Comment:
nice! was this generated? Can we provide a Postman collection?

Reply:
No, I formatted this on my own haha

Reply:
ouch - we should automate this next time, related to #160 I guess